src/util/nft.js

```diff
@@ -298,13 +298,20 @@ export function formatNFTClassInfo(classData) {
 
 export const fetchAllNFTClassFromChain = async (
   axios,
-  { iscnOwner, nftOwner }
+  { iscnOwner, nftOwner, nocache = false }
 ) => {
+  const params = {
+    iscnOwner,
+    nftOwner,
+  };
+  if (nocache) {
+    params.nocache = nocache;
```

Comment:
seems the behaviour would be the same if we just pass whole params in with `nocache` no matter true or false?